app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -3288,6 +3304,9 @@ class ChatController(args: Bundle) :
         private const val VOICE_RECORD_CANCEL_SLIDER_X: Int = -50
         private const val VOICE_MESSAGE_META_DATA = "{\"messageType\":\"voice-message\"}"
         private const val VOICE_MESSAGE_FILE_SUFFIX = ".mp3"
+        private const val FILE_DATE_PATTERN = "yyyy-MM-dd HH-mm-ss"
+        private const val VIDEO_PREFIX_PART = "Talk Video "
```

Comment:
i keep it simple and removed "Talk", so prefix is only "Video "

Reply:
- for photos we only have only the date + suffix (we should change this when photo camera is replaced i suggest)
- for voice messages we have a translation (where "Talk recording from" is also fine for branded clients i guess, because it#s a recording of a talk..)
- on iOS there are just "random" numbers for the filenames. (at least they're random to the user. maybe this can be changed in the future @Ivansss )

so yeah i think it makes sense to begin with some file naming strategy here. will make it translatable without to mention the product "Talk"